languages/c/reference/README.md

```diff
@@ -43,7 +43,7 @@ The C concept exercises are based on language concepts. The list below contains
 
 #### Patterns
 
-- [Callback](https://en.wikipedia.org/wiki/Callback_(computer_programming))
+- [Callback](<https://en.wikipedia.org/wiki/Callback_(computer_programming)>)
```

Comment:
Don't think this is right...
```suggestion
- [Callback](https://en.wikipedia.org/wiki/Callback_(computer_programming))
```

Reply:
It is. The reason for the `<` and `>` is due to the fact that the URL itself uses parentheses.